components/Settings.js

```diff
@@ -171,109 +160,102 @@ const MiscSettings = React.memo(({ format, reset }) => {
 
 const MenuButton = React.memo(({ name, select, selected }) => {
   return (
-    <button onClick={select(name)} className={selected === name ? 'selected' : ''}>
-      {name}
-      <div className="arrow-icon">
-        <Arrows.Right />
-      </div>
+    <div className="menu-button">
+      <Button
+        padding="8px"
+        onClick={select(name)}
+        background={selected === name ? COLORS.BLACK : COLORS.DARK_GRAY}
+      >
+        {name}
+        <div className="arrow-icon">
+          <Arrows.Right />
+        </div>
+      </Button>
       <style jsx>
         {`
-          button {
-            outline: none;
-            border: none;
-            background: transparent;
-            color: ${COLORS.SECONDARY};
+          .menu-button {
             display: flex;
-            padding: 8px;
             height: 33px;
-            line-height: 0;
-            cursor: pointer;
-            font-size: 14px;
             border-bottom: 1px solid ${COLORS.SECONDARY};
             position: relative;
             align-items: center;
           }
 
-          button:last-child {
+          .menu-button:last-child {
             ${selected === 'Window' ? '' : 'border-bottom: none;'};
           }
 
-          button.selected {
-            background-color: ${COLORS.BLACK};
-          }
-
           .arrow-icon {
             position: absolute;
             right: 14px;
             top: 11px;
           }
         `}
       </style>
-    </button>
+    </div>
+  )
+})
+
+const removeButtonStyles = {
+  position: 'absolute',
+  top: '6px',
+  right: '6px',
+  width: '11px',
+  height: '11px',
+  borderRadius: '999px'
+}
+
+const Preset = React.memo(({ remove, apply, selected, preset }) => {
```

Comment:
While you are at it, can you move the Preset menu into its own file? 🙂

Reply:
Yes agreed I was wanting to do that. I did that for the new Theme feature and it made everything much better. I planned to wait until after the Theme work is done though. This PR is mostly things I pulled out of that work.